cmd/nerdctl/pull.go

```diff
@@ -92,6 +102,59 @@ func pullAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if isVerify, err := cmd.Flags().GetString("verify"); err == nil && isVerify == "cosign" {
+
+		ref, err := name.ParseReference(rawRef)
+		if err != nil {
+			logrus.Errorf("Unable to parse reference: %v\n", err)
+		}
+
+		digest, err := remote.ResolveDigest(ref)
+		if err != nil {
+			logrus.Errorf("Unable to resolve digest %q: %v\n", ref.String(), err)
+		}
+
+		logrus.Debugf("verifying image: %s\n", digest)
```

Comment:
the digest doesn't seem to be passed to cosign nor `imgutil.EnsureImage`

Reply:
thanks, done